### PR TITLE
Update packages with Qowaiv dependency

### PR DESCRIPTION
**Pull request by Corniel**
Both `Qowaiv.Validation.Fluent` and `Qowaiv.Validation.DataAnnotations` have a depedency on `Qowaiv`. These depdencies should be up-to-date.

### CODE REVIEW

File: src/Qowaiv.Validation.Fluent/Qowaiv.Validation.Fluent.csproj [hunk 1]
```diff
@@ -4,12 +4,14 @@
   
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;net5.0;net6.0</TargetFrameworks>
```
Comment (krakkepit) [inline src/Qowaiv.Validation.Fluent/Qowaiv.Validation.Fluent.csproj:6]: Shouldn't we show the supported frameworks in a LIFO order? Like they did in https://docs.microsoft.com/en-us/dotnet/standard/frameworks

File: src/Qowaiv.Validation.Fluent/Qowaiv.Validation.Fluent.csproj [hunk 2]
```diff
@@ -4,12 +4,14 @@
...
-    <Version>0.1.1</Version>
+    <Version>0.2.0</Version>
     <PackageReleaseNotes>
+v0.2.0
+- Updated Qowaiv depedency.
```
Comment (krakkepit) [inline src/Qowaiv.Validation.Fluent/Qowaiv.Validation.Fluent.csproj:10]: Typo: depeNdency

Comment (Corniel) [reply src/Qowaiv.Validation.Fluent/Qowaiv.Validation.Fluent.csproj]: Oops.

File: src/Qowaiv.Validation.Fluent/Qowaiv.Validation.Fluent.csproj (hunk 1 above)
Comment (Corniel) [reply src/Qowaiv.Validation.Fluent/Qowaiv.Validation.Fluent.csproj:6]: I don't think that common or making any difference. This is just the config for the compiler to know to which target frameworks to compile (and to pack for NuGet).

Comment (krakkepit) [reply src/Qowaiv.Validation.Fluent/Qowaiv.Validation.Fluent.csproj:6]: Fine by me!

**Review by krakkepit — APPROVED** [review]

**Pull request merged**